src/features/conversations/components/ConversationDetail/ConversationDetail.tsx

```diff
@@ -1,12 +1,98 @@
-// import { useParams } from "react-router-dom";
+import { useState} from 'react';
+import Input from '../Input';
 
 export default function ConversationDetail() {
-  //   const { conversationId } = useParams();
+  const [messages, setMessages] = useState<any[]>([]); 
```

Comment:
- Message là một entity phải có type rõ ràng không dùng any, tạm thời BE chưa define thì lấy từ ERD.
- Get detail conversation từ api để get list message rồi truyền vào state messages.
- Lúc user gửi message thì thực hiện optimistic update rồi call api send message 
rồi lấy data từ response để  gen ra conent của chatbot reply.
- Suy nghĩ thử làm mock cho phần gửi message, update conversation thế nào và apply tanstack-query để call api.

```suggestion
  const [messages, setMessages] = useState<IMessage[]>(); 
```